docs/source/_rst/tutorials/tutorial5/tutorial.rst

```diff
@@ -23,6 +23,9 @@ First of all we import the modules needed for the tutorial. Importing
     if IN_COLAB:
       !pip install "pina-mathlab"
       !pip install scipy
+      # get the data
+      !wget https://github.com/mathLab/PINA/blob/master/tutorials/tutorial5/Data_Darcy.mat
```

Comment:
You should download the RAW file, that link takes the Github web page containing the file

Reply:
I'm checking with Colab and it actually works with this syntax

Reply:
The error is about the folder containing the .mat files, indeed it has been called "dat" and not "data"